src/main.rs

```diff
@@ -30,8 +30,9 @@ fn main() {
     let app = App::new("RustPython");
     let matches = parse_arguments(app);
     let settings = create_settings(&matches);
-    let vm = VirtualMachine::new(settings);
+    let mut uninitialized_vm = VirtualMachine::new(settings);
 
+    let vm = uninitialized_vm.initialize(true);
```

Comment:
You can just do
```rust
let vm = VirtualMachine::new(settings);
vm.initialize(cfg!(not(target_os = "wasi")));
```

Reply:
You could add a setting to the `Settings` object, having a boolean to initialize the vm yes or no.